packages/ui-time-select/src/TimeSelect/props.ts

```diff
@@ -82,73 +50,70 @@ type TimeSelectProps = TimeSelectOwnProps &
     InputHTMLAttributes<TimeSelectOwnProps>
   >
 
-const propTypes: PropValidators<PropKeys> = {
+type TimeSelectOwnProps = {
   /**
    * The form field label.
    */
-  renderLabel: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
+  renderLabel: React.ReactNode | (() => React.ReactNode)
   /**
    * Whether to default to the first option when `defaultValue` hasn't been specified.
    */
-  defaultToFirstOption: PropTypes.bool,
+  defaultToFirstOption?: boolean
   /**
    * An ISO 8601 formatted date string representing the current selected value. If defined,
    * the component will act controlled and will not manage its own state.
    */
-  value: controllable(I18nPropTypes.iso8601, 'onChange'),
+  value?: string // TODO: controllable(I18nPropTypes.iso8601, 'onChange'),
   /**
    * An ISO 8601 formatted date string to use if `value` isn't provided.
    */
-  defaultValue: I18nPropTypes.iso8601,
+  defaultValue?: string
   /**
    * The id of the text input. One is generated if not supplied.
    */
-  id: PropTypes.string,
+  id?: string
   /**
    * The format to use when displaying the possible and currently selected options.
    *
    * See [moment](https://momentjs.com/docs/#/displaying/format/) for the list
    * of available formats.
    */
-  format: PropTypes.string,
+  format?: string
   /**
    * The number of minutes to increment by when generating the allowable options.
    */
-  step: PropTypes.oneOf([5, 10, 15, 20, 30, 60]),
+  step?: 5 | 10 | 15 | 20 | 30 | 60
   /**
    * Specifies if interaction with the input is enabled, disabled, or readonly.
    * When "disabled", the input changes visibly to indicate that it cannot
    * receive user interactions. When "readonly" the input still cannot receive
-   * user interactions but it keeps the same styles as if it were enabled.
+   * user interactions, but it keeps the same styles as if it were enabled.
    */
-  interaction: PropTypes.oneOf(['enabled', 'disabled', 'readonly']),
+  interaction?: 'enabled' | 'disabled' | 'readonly'
   /**
    * Html placeholder text to display when the input has no value. This should
    * be hint text, not a label replacement.
    */
-  placeholder: PropTypes.string,
-  /**
-   * Whether or not the text input is required.
-   */
-  isRequired: PropTypes.bool,
+  placeholder?: string
+  isRequired?: boolean
```

Comment:
The comment is missing for `isRequired` but I guess it is pretty obvious what this prop does :)